lib.nix

```diff
@@ -2,8 +2,13 @@
 # nix-repl lib.nix
 
 let
-  hostPkgs = import <nixpkgs> {};
-  lib = hostPkgs.lib;
+  fetchNixPkgs = let try = builtins.tryEval <nixops_pkgs>;
+    in if try.success
+    then builtins.trace "using host <nixops_pkgs>" try.value
+    else builtins.trace "fetching nixpkgs" (import ./fetch-nixpkgs.nix);
```

Comment:
I think in `else` case we can leave out the trace since it's a desired operation.